docs/FAQ.md

```diff
@@ -64,6 +64,7 @@ OpenCost has core billing integrations with:
 - AWS/EKS
 - Azure/AKS
 - GCP/GKE
+- OCI/OKE
```

Comment:
Leave as `OCI/OKE` or change to `Oracle Cloud Infrastructure(OCI)/OKE` to avoid confusion with oci term?